### PR TITLE
Fixes #2873 - Add AB testing plumbing

### LINKED ISSUE #2873: Add support for AB testing in webcompat.com
As part of our (Open Innovation) experiments to test some UX changes in the reporting flow, it would be useful to be able to use some sort of AB testing capabilities in webcompat.com.

### PR DESCRIPTION
**Pull request by johngian**
This PR fixes issue #2873
## Proposed PR background

Adds lightweight AB testing capabilities in webcompat.com backend.

   * [x]  I have read the [Pull Request + Code Style Guidelines](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md) thoroughly.

### CODE REVIEW

**Comment by johngian** [comment]
@karlcow Hey! This is the backend AB implementation we discussed in all hands. Let me know what you think.

File: docs/ab-testing.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+```
+
+## How it works
+On each request, the AB helper randomly selects a number and if the number falls within the range
+of the experiment variation, user participates in this experiment's variation. This is implemented
+by adding setting a cookie with name `<experiment_id>` and value `<variation>`. Each experiment
+expires after `<max-age>` seconds.
+
+## Usage
```
Comment (karlcow) [inline docs/ab-testing.md]: Could you include an example, steps of what someone should do in case they need to implement an experiment in the code.

**Comment by johngian** [comment]
@karlcow PR updated

**Comment by johngian** [comment]
@karlcow I accidentally pushed my local docker env commits (that might be useful but in a different PR) and I instantly reverted them. Are you sure you are checking the right revision?

**Comment by karlcow** [comment]
Perfect. Cool. Thanks.

**Pull request merged**